frontend/integration-tests/tests/kubevirt/clone.vm.scenario.ts

```diff
@@ -0,0 +1,230 @@
+/* eslint-disable no-undef, max-nested-callbacks */
+import { execSync } from 'child_process';
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+
+// eslint-disable-next-line no-unused-vars
+import { removeLeakedResources, waitForCount, searchYAML, VM_BOOTUP_TIMEOUT, PAGE_LOAD_TIMEOUT } from './utils';
+import { appHost, testName } from '../../protractor.conf';
+import { filterForName, isLoaded, resourceRowsPresent, resourceRows } from '../../views/crud.view';
+import { basicVmConfig, networkInterface, testNad, getVmManifest, hddDisk, cloudInitCustomScriptConfig } from './mocks';
+import * as wizardView from '../../views/kubevirt/wizard.view';
+import Wizard from './models/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+import { nicsTab, disksTab } from '../../views/kubevirt/virtualMachine.view';
+
+
+describe('Test clone VM.', () => {
+  const wizard = new Wizard();
+  const testContainerVm = getVmManifest('Container');
+  const vm = new VirtualMachine(testContainerVm.metadata.name, testContainerVm.metadata.namespace);
+
+  describe('Test Clone VM wizard dialog.', () => {
+    beforeAll(async() => {
+      execSync(`echo '${JSON.stringify(testContainerVm)}' | kubectl create -f -`);
+      execSync(`echo '${JSON.stringify(testNad)}' | kubectl create -f -`);
+    });
+
+    afterAll(async() => {
+      execSync(`kubectl delete -n ${testName} net-attach-def ${testNad.metadata.name}`);
+      execSync(`kubectl delete -n ${testName} vm ${testContainerVm.metadata.name}`);
+    });
+
+    it('Display warning in clone wizard when cloned vm is running.', async() => {
+      await vm.action('Start');
+      await vm.action('Clone');
+
+      await browser.wait(until.visibilityOf(wizardView.cloneVmWarning), PAGE_LOAD_TIMEOUT);
+
+      await wizard.close();
+      await vm.action('Stop');
+    }, VM_BOOTUP_TIMEOUT);
+
+    it('Prefill correct data in the clone VM dialog.', async() => {
+      await vm.action('Clone');
+
+      expect(wizardView.cloneVmWarning.isPresent()).toBe(false, 'Warning should not be present.');
+      expect(wizardView.nameInput.getAttribute('value')).toEqual(`${vm.name}-clone`, '\'-cloned\' should be appended to cloned VM name.');
+      expect(wizardView.descriptionInput.getText()).toEqual(testContainerVm.metadata.annotations.description, 'Description should match original VM description.');
+      expect($(wizardView.namespaceDropdownId).getText()).toEqual(vm.namespace, 'Namaspace should match original VM namespace.');
+
+      await wizard.close();
+    });
+
+    it('Validate VM name.', async() => {
+      await vm.action('Clone');
+
+      expect(wizardView.cloneVmWarning.isPresent()).toBe(false, 'Warning should not be present.');
+      await wizard.fillName(vm.name);
```

Comment:
Can you also check that VM name is validated when namespace is changed ? ie Im trying to clone `vmname1` and I will change namespace to `namespace1`.  `namespace1` already has `vmname1` -> error is shown

Reply:
Done.